src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -0,0 +1,76 @@
+package seedu.duke.parser;
+
+import seedu.duke.ui.UI;
+import seedu.duke.exception.DukeException;
+import seedu.duke.task.*;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Parser {
+    public void parse(String command, UI ui, TaskList tasks, String filePath){
+        try {
+            FileWriter fw = new FileWriter(filePath, true);
```

Comment:
Single-responsibility Principle: A class should have one and only one reason to change, meaning that a class should have only one job.

Reply:
Oh right. Okay thank you! I will implement that.